.babelrc

```diff
@@ -1,7 +1,11 @@
 {
-  "env": {
-    "test": {
-      "plugins": ["transform-es2015-modules-commonjs"]
-    }
-  }
+  "presets": [
+    "es2015",
```

Comment:
Tried using `babel-preset-env` and the build is still 2.24MB. I suspect it because of the [browser versions targeted](https://github.com/guardian/media-atom-maker/commit/24b1d8fc572daa38b147bcc59a004961e62e3f14); this list was provided from ESD.